IO-phone/src/app/register/model.ts

```diff
@@ -7,6 +7,12 @@ export enum LobbyEventStatus {
     UNKNOWN_ERROR
 }
 
+export const LobbyEventMessages = {
+    SUCCESS: "registered successfully",
+    START: "start",
+    USER_KICKED: "kicked"
+}
```

Comment:
Coś takiego jest stałą, ale odnosi się to tylko do referencji. Niestety coś takiego:
```
const LobbyEventMessages = {
    SUCCESS: "registered successfully",
    START: "start",
    USER_KICKED: "kicked"
}

LobbyEventMessages.START = "whatever"

console.log(LobbyEventMessages.START)
```
Zwraca:
```
[LOG]: "whatever" 
```

Fix jest bardzo prosty, trzeba dodać na końcu `as const`:

```
const LobbyEventMessages = {
    SUCCESS: "registered successfully",
    START: "start",
    USER_KICKED: "kicked"
} as const
```

Możesz sobie zobaczyć, że wcześniej `LobbyEventMessages.SUCCESS` było typu `string`, a teraz: `SUCCESS: "registered successfully"` - ma tylko jedną możliwą wartość

Reply:
Masz rację, umknęło mi to. Dzięki za spotrzegawczość